README.md

```diff
@@ -1 +1,37 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microverse Project
+
+> Built this project as part of the Microverse code reviewing exercise and training.
+
+## Built With
+
+- HTML, CSS
+- Github flow
+- VSCode
+
```

Comment:
- [ ] Kindly add the Getting started section from the Microverse ReadMe template, in it you should add steps to clone the repo since most people who view your projects might not have the technical expertise to do so,

Reply:
@zhadier Below is a screenshot of the Getting Started section 
<img width="960" alt="getting started" src="https://user-images.githubusercontent.com/60811467/177331462-792c6823-4229-4f0d-91b1-3468de4465ab.png">